tests/unit/semaphore_test.cc

```diff
@@ -249,16 +249,18 @@ SEASTAR_TEST_CASE(test_with_semaphore) {
     return do_with(semaphore(1), 0, [] (semaphore& sem, int& counter) {
         return with_semaphore(sem, 1, [&counter] {
             ++counter;
-        }).then([&counter, &sem] () {
+            BOOST_REQUIRE_EQUAL(counter, 1);
+        })/*.then([&counter, &sem] () {
```

Comment:
Why is it commented out?

Reply:
ouch, I was debugging some failure here and committed the change by mistake.